### PR TITLE
<TreeView /> Styling

### PR DESCRIPTION
**Pull request by liorzisman**
Resolves issue #120

### CODE REVIEW

**Comment by liorzisman** [comment]
Also added className prop according to requirement

File: src/components/tree-view/tree-view.tsx [hunk 1]
```diff
@@ -102,7 +103,7 @@ export class TreeView extends React.Component<TreeViewProps, {}>{
     render() {
         const TreeNode = this.props.itemRenderer!;
         return (
-            <div data-automation-id='TREE_VIEW' className="tree-view">
+            <div data-automation-id='TREE_VIEW' className={'tree-view' + this.props.className ? ` ${this.props.className}` : ''}>
```
Comment (tobich) [inline src/components/tree-view/tree-view.tsx]: please use the root() util to merge in this.props.className

Comment (liorzisman) [inline src/components/tree-view/tree-view.tsx]: done

**Comment by liorzisman** [comment]
@giladgrush @tobich ready for review

File: src/components/tree-view/tree-view.st.css [hunk 2]
```diff
@@ -1,26 +1,51 @@
...
+}
+
 .root {
-    font-family: 'Helvetica', Arial, sans-serif;
-    font-size: 18px;
+    font-family: Arial, sans-serif;
+    font-size: 16px;
     line-height: 1.5;
     text-align: left;
+    color: #000000;
```
Comment (AviVahl) [inline src/components/tree-view/tree-view.st.css]: use `black` :P

File: src/components/tree-view/tree-view.st.css [hunk 3]
```diff
@@ -1,26 +1,51 @@
...
 }
 
 .tree-node {
     -st-states: selected, focused;
     cursor: pointer;
-    display: inline;
-    padding: 2px 5px;
+    display: inline-block;
+    padding: 3px 7px 5px;
+    width: 228px;
```
Comment (AviVahl) [inline src/components/tree-view/tree-view.st.css]: why is there a static width set?

File: src/components/tree-view/tree-view.st.css (hunk 2 above)
Comment (liorzisman) [inline src/components/tree-view/tree-view.st.css]: will fix

File: src/components/tree-view/tree-view.st.css (hunk 3 above)
Comment (liorzisman) [inline src/components/tree-view/tree-view.st.css]: styling according to zeplin. all nodes had the same width regardless of text length...

Comment (tyv) [inline src/components/tree-view/tree-view.st.css]: I would consider that as a reason to open discussion. 
Usually I expect tree nodes to be as long as my text is there.

Comment (liorzisman) [inline src/components/tree-view/tree-view.st.css]: Talked with Gilaad to fully understand the style. It seems this is the proposal for the default item renderer (the item's background going until the edge of the container).

**Comment by liorzisman** [comment]
approved by avi

**Pull request merged**